### ISSUE TITLE
/bin/sh: sdkmanager: command not found

### ISSUE THREAD

**Issue by SDGGiesbrecht** [ReactiveCircus/android-emulator-runner#123]
For the last day or two, all actions have been failing with this message:

```
Installing latest build tools, platform tools, and platform.
/bin/sh -c \sdkmanager --install 'build-tools;30.0.3' platform-tools 'platforms;android-29' > /dev/null
/bin/sh: sdkmanager: command not found
/Users/runner/Library/Android/sdk/platform-tools/adb -s emulator-5554 emu kill
error: could not connect to TCP port 5554: Connection refused
```

As recently as four days ago, actions were successful, and logs appeared as follows. Note the additional first step.

```
Installing new cmdline-tools.
/usr/bin/unzip -q /Users/runner/work/_temp/1f68391f-809c-4848-9a14-8eca0e8ed308
Installing latest build tools, platform tools, and platform.
/bin/sh -c \sdkmanager --install 'build-tools;30.0.3' platform-tools 'platforms;android-29' > /dev/null
Installing latest emulator.
/bin/sh -c \sdkmanager --install emulator > /dev/null
[...]
```

I can see from your repository’s main page that the latest release was 8 days ago. That suggests to me that the change was not in your code, but with the underlying GitHub’s host. Maybe they added a partial installation of the command line tools, which tricks your action into believing it already has everything it needs even though it actually doesn’t?

**Comment by ychescale9** [comment]
Can you try to ping the android-emulator-runner version to `v2.13.0`?

Can you share your workflow and a full log?

**Comment by SDGGiesbrecht** [comment]
> Can you try to ping the android-emulator-runner version to v2.13.0?

I’ll try and get back to you.

> Can you share your workflow and a full log?

I doubt they will help, but here is one such [workflow](https://github.com/SDGGiesbrecht/SDGCornerstone/blob/ac3981b7b0177de2f33c690d47156bb575c9a52d/.github/workflows/Workspace%20Validation.yaml#L624-L629) and [corresponding log](https://github.com/SDGGiesbrecht/SDGCornerstone/pull/399/checks?check_run_id=1679177337).

**Comment by SDGGiesbrecht** [comment]
> Can you try to ping the android-emulator-runner version to v2.13.0?

Here are the [commit](https://github.com/SDGGiesbrecht/SDGCornerstone/commit/12284b2efc8e1589a429fd124982eaa0fa7e26c2#diff-f01c595dfd972116fb04a0c423935f00695471522d6bd96af637b44dfef4302cR108) and [log](https://github.com/SDGGiesbrecht/SDGCornerstone/runs/1692638449).

**Comment by ychescale9** [comment]
Ah I see. You're using [macos-11](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11.0-Readme.md#android) which has the `cmdline-tools` installed so this action skips the initial downloading and configuration of the cmdline-tools, which also skips adding the path for `sdkmanager` etc.

I'm fixing it now.

**Comment by ychescale9** [comment]
`macos-11.0` should be supported now. Let me know if it works for you :)

**Comment by SDGGiesbrecht** [comment]
Yes, it works now. Thanks.

**Comment by s-ayush2903** [comment]
@ychescale9 We're still getting the same problem even on updating the runner version to `2.13.0`, can you help us with that. 
Run: https://github.com/kiwix/kiwix-android/pull/2562/checks?check_run_id=1783849832
Commit(that updates rc version): https://github.com/kiwix/kiwix-android/pull/2562/commits/574cca8f33d474da824a488da12cc10938314b46

Also to mention that our nightly build is also failing since last two days, the reason being the same, yeah I admit that we haven't runner version in nightly we've done so in our coverage reporting but still they're failing, can you please look into it

**Comment by ychescale9** [comment]
The fix is available in the latest release `2.14.2`. You can also use the latest version instead of pinging to a specific version to get the latest update: `ReactiveCircus/android-emulator-runner@v2`.

**Comment by s-ayush2903** [comment]
Thanks for help, it fixed all failures! :smiley:
